### PR TITLE
Check all commits in branch for examples (to trigger testing)

### PR DESCRIPTION
**Pull request by sftim**
In #14147 @tengqm pointing out that that PR should have _failed_ CI testing for not adding the changed examples to a list of test cases. It didn't.

This currently-draft PR is my take on a partial fix: it looks at the list of files changed on the current branch (since it diverged `master`) and checks if any of those are in any locale's `examples` folder.

At least, that's what I think it does.

/sig testing

### CODE REVIEW

**Comment by tengqm** [comment]
Code improvement like this is fine. We have another problem regarding the test script -- after replacing travis build with netlify, the script won't work because some environment variables we had on travis are not available on netlify.

**Comment by sftim** [comment]
I'll mark this ready to review anyway

File: scripts/test_examples.sh [hunk 1]
```diff
@@ -3,21 +3,18 @@
...
-FILES=`git log -n 2 --name-only --format=""`
+FILES=($( git diff "$( git merge-base --fork-point master )" --name-only ))
 
 TEST_EXAMPLES=No
 
-# Currently examine en directory only, can extend to other lang when neded
-for f in $FILES; do
-  if [[ $f =~ "content/en/examples/" ]]; then
+# Check if examples folders (all locales) change in this branch
+if printf -- '%s\n' "${FILES[@]}" | grep -qE '^"?content/[^/]+/examples/'; then
```
Comment (DanyC97) [inline scripts/test_examples.sh]: nice one, like this bit 👍

**Pull request merged**